roofit/hs3/src/JSONIOUtils.h

```diff
@@ -0,0 +1,10 @@
+#ifndef JSONIOUtils_h
+#define JSONIOUtils_h
+
+#include "JSONIOUtils.h"
+
+#include <string>
+
+bool endsWith(std::string_view str, std::string_view suffix);
```

Comment:
This `endsWith` function is also used in `JSONFactories_HistFactory.cxx`. Can you include this utils header there too? Then the `endsWith` function doesn't need to get re-declared in `JSONFactories_HistFactory.cxx`.